### DISCUSSION TITLE
Controling an RPi pigpiod using Windows

### DISCUSSION THREAD

**Discussion by Ironarrow78** [mattjlewis/diozero#128] [Q&A] [answered]
Hi,

is there any sample application how to controll the pigpiod from a RPi using a Windows PC in a plain way?

I mean, I created a spring boot web application (for REST apis) and I added the pom.xml dependency diozer-core (from mvnrepository.com). 
Is there any application.properties value or new LED(17, ??????) parameter to set the IP of the remote pigipod so I can send commands directly from my JAVA application?

Thank you very much in advance,

Rainer

**Comment by mattjlewis** [comment, marked as answer]
Yes, you need to add the pigpioj provider to the class path. See here for an example: https://www.diozero.com/concepts/remotecontrol.html#pigpio

**Comment by Ironarrow78** [comment]
Thank you very much. 
Adding the diozero-provider-pigpio dependency to the pom.xml and setting the PIGPIOD_HOST environment variable and it works.

Best regards,
Rainer
